docs/modules/ROOT/pages/how-tos/configuring/referencing-secrets-in-Containerfile.adoc

```diff
@@ -0,0 +1,49 @@
+= Referencing Secrets in a Containerfile
+
+There may be times when you need to reference a secret directly in your Containerfile. For example, if your build uses cryptographic parameters stored in secrets. This can be accomplished by using the `ADDITIONAL_SECRET` link:https://github.com/konflux-ci/build-definitions/blob/main/task/buildah-oci-ta/0.4/README.md[param] parameter of the `build-container` task.
+
+.Procedure
+
+. Create the needed secret using link:https://konflux-ci.dev/docs/how-tos/configuring/creating-secrets/[secrets].
+. Set the value of the `ADDITIONAL_SECRET` parameter to the name of the secret that you want to use in your Containerfile.
+. In the Containerfile, use a `RUN` command to mount the secret and make it available for use.
+
+== Example: Setting the ADDITIONAL_SECRET Parameter in the build-container Task
+
+Let's use an example of a cryptographic secret used to customize encryption in a Containerfile.
+
+Given that you have created a secret named `cryptography` that contains various cryptography-related values (such as salts), here is how you would pass it to the `ADDITIONAL_SECRET` parameter in the build-container task in your Tekton pipelines:
+
+[source,yaml]
+----
+...
+  tasks:
+    - name: build-container
+      params:
+        - name: ADDITIONAL_SECRET
+          value: "cryptography"
+...
+----
+
+== Example: Referencing the ADDITIONAL_SECRET in Your Containerfile Using RUN
+
+Building on the example above, we want to use these secrets as environment variables for a Rust cargo build command. We do this by mounting the secrets using the `RUN` command, then setting them as environment variables for the build process. Note that the `cryptography` secret contains two values: `SALT` and `KEY_HASH`. This example shows how to reference them both:
```

Comment:
Instead of this note about the secret, can you provide an example secret?

Reply:
Ok added!